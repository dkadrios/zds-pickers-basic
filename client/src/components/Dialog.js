import React from 'react'
import MuiDialog from '@material-ui/core/Dialog'
import Slide from '@material-ui/core/Slide'

/* istanbul ignore next */
const Transition = props => (
  <Slide
    direction="up"
    {...props}
  />
)

const Dialog = props => (
  <MuiDialog
    TransitionComponent={Transition}
    {...props}
  />
)

Dialog.propTypes = {}

export default Dialog
