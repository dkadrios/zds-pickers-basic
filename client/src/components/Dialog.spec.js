import React from 'react'
import Dialog from './Dialog'

const props = { open: true }

describe('Dialog tests', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Dialog {...props}>content</Dialog>)
    expect(wrapper).toMatchSnapshot()
  })
})
