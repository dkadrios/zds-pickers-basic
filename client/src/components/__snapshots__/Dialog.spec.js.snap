// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog tests renders correctly 1`] = `
<WithStyles(Dialog)
  TransitionComponent={[Function]}
  open={true}
>
  content
</WithStyles(Dialog)>
`;
